api/lib/controllers/meeting.js

```diff
@@ -229,7 +229,27 @@ module.exports = {
   async index( req, res ) {
     const subject_email = req.credentials.user.email;
     const subject_id = req.credentials.sub;
-    const { limit = 0, skip = 0 } = req.query; // Thanks tom
+    const {  filters = `{ "owners": [], "name": ""}` } = req.query; // Thanks tom
+    const { limit = 0, skip = 0 } = req.query; //will need to change these for total count
+
+    const { owners, name } = JSON.parse( filters );
+
+    const pipelineFilters = [];
+
+    owners.length && pipelineFilters.push({
+      $match:  {
+        owner_id: {
+          $in: owners.map( ( owner ) => ObjectID( owner ) )
+        }
+      }
+    });
+
+    name && pipelineFilters.push({
+      $match: { name: { $regex: name, $options: 'i' } }
+    });
```

Comment:
Thoughts on making the regex match for any part of the name: `*${name}*`.

It also probably doesn't make sense to keep the participant lookup in the query if the owner filter is the owners id.

Reply:
not sure what you mean, there will be other filters eventually and the participant lookup looks up if the user's email is a participant in any meetings

Reply:
You can ignore the commend about the regex.

As for the second comment, I was thinking that if a user is filtering by their own `owner_id` then the participant lookup would be a waste. But if you are saying that they could filter by multiple owners then yeah, it doesn't make sense to try to optimize for owned meetings by conditionally adding the participant lookup.